JASP-Desktop/analysis/jaspcontrolbase.cpp

```diff
@@ -49,6 +49,28 @@ void JASPControlBase::setHasWarning(bool hasWarning)
 	}
 }
 
+void JASPControlBase::setRunOnChangeToChildren(bool change)
```

Comment:
Is it hard to get the children to simply check their parent? Instead of having to go through the children and changing it manually?
Would be a bit cleaner right? And you could simply connect the runOnChangeChanged signals from parent->child.

Reply:
Although I guess this isn't so bad either and we could reuse the code you've written now (getChildJASPControls) for the QML help file generation as well. So nvm.